doc/configuration.rst

```diff
@@ -1178,10 +1178,11 @@ Implements:
 
 .. code-block:: yaml
 
-   NetworkUSBStorageDriver: {}
+   NetworkUSBStorageDriver:
+     image: myusb.image
```

Comment:
This does not work. Either
`filename = self.target.env.config.resolve_path(self.image)` -> specify image filename directly
or
`filename = self.target.env.config.get_image_path(self.image)` -> specify "images subkey" and add "images" to example config.